src/main/resources/app.conf

```diff
@@ -347,34 +381,45 @@ app {
         # the initial delay before retrying that will be multiplied with the 'factor' for each attempt
         initial-delay = ${app.key-value-store.retry.initial-delay}
         initial-delay = ${?REALMS_KEY_VALUE_STORE_RETRY_INITIAL_DELAY}
+        # the maximum delay applied to the retrying strategy
+        max-delay = ${app.sourcing.retry.max-delay}
```

Comment:
update value to: `${app.key-value-store.retry.max-delay}`
and add env override: `REALMS_KEY_VALUE_STORE_RETRY_MAX_DELAY`